### ISSUE TITLE
Feature: Generate OAuth read-only (and read-write) tokens for server apps

### ISSUE THREAD

**Issue by DougReeder** [remotestorage/armadietto#75]
Other servers that access RS require setting the tokens manually, as they generally don't have a Web interface.

This is also required to run the RS api-test-suite.

This would probably be part of an admin UI.

**Comment by raucao** [comment]
What if we just wrote a simple unhosted app that can aquire custom tokens for server integration? Since servers are not bound by Origin restrictions (they can just send whatever origin they want), the client ID and redirect URI may as well be a known app that creates tokens for those.

**Comment by DougReeder** [comment]
A new client app? That sounds straightforward.

**Comment by raucao** [comment]
Yeah, it would be similar to https://github.com/raucao/rs-backup-auth-page but with a form to specify category name (or root access) and permission level.
